timeout 3 sudo python /home/pi/api/getGps.py